.gitignore

```diff
@@ -79,3 +79,4 @@ cover
 
 .venv/
 venv/
+*.json
```

Comment:
do we really want to ignore all `json` files?

Reply:
No, sorry